LICENSE.txt

```diff
@@ -2252,3 +2252,10 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
+
+--------------------------------------------------------------------------------
+java/vector/src/main/java/org/apache/arrow/vector/util/IntObjectHashMap.java
```

Comment:
Maybe worth to mention in NOTICE file too.

Reply:
If Netty doesn't have `NOTICE` contents for this file, we don't need to change our `NOTICE` file.
See also: https://infra.apache.org/licensing-howto.html#alv2-dep